docs/schematypes.jade

```diff
@@ -20,14 +20,14 @@ block content
     The following are all the valid SchemaTypes in mongoose.
 
     - [String](#strings)
-    - [Number](./api.html#schema-number-js)
+    - [Number](./api.html#Number)
```

Comment:
I'd recommend this link to a `#numbers` section in this file that describes the particulars of the Mongoose number type, like the `#strings` section. The intent with this issue is to add a section that describes what gets casted to a number, what custom validators are available for numbers, and what passes the `required` validator for numbers. Maybe in the future we add notes on how `$inc` works with numbers. IMO the particulars of numbers are best explained in prose rather than API docs, because the end user generally doesn't call the functions in the `schema/number` API. Does that make sense?

Reply:
Yeah, I'll do this instead :)